process/project-lifecycle.md

```diff
@@ -0,0 +1,159 @@
+# Project Life Cycle
+
+Projects are either directly under the governance of the Technical Advisory Committee (TAC) or a specific Working Group (WG). The progression of a project lies accordingly within the TAC or the specific WG the project reports to (i.e., "the parent WG").
```

Comment:
The TAC should be described as the responsible party -- a WG can recommend a project move to specific maturity levels, but the TAC must approve it. 

This provides consistency to all external parties who will rely on the OpenSSF *as a whole* to signal in a consistent manner, and it aligns with the Charter's definition of the TAC's role.

Proposed wording:

```
Projects are either directly under the governance of the Technical Advisory Committee (TAC) or a specific Working Group (WG). The TAC is responsible for reviewing and approving changes to all Project's Life Cycles. Where a Project reports into a specific WG, that WG can support the Project's progression and provide recommendations to the TAC.

Reply:
nit: s/approving changes/approving significant changes
I don't think we would have time to look at all changes.

Reply:
Done.